src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt

```diff
@@ -137,6 +140,47 @@ object SonarLintIntelliJClient : SonarLintRpcClientDelegate {
         suggestionsByConfigScopeId.forEach { (configScopeId, suggestions) -> suggestAutoBind(findProject(configScopeId), suggestions) }
     }
 
+    override fun suggestConnection(suggestionsByConfigScope: Map<String, List<ConnectionSuggestionDto>>) {
+        for (suggestion in suggestionsByConfigScope) {
+            val project = BackendService.findModule(suggestion.key)?.project
+                ?: BackendService.findProject(suggestion.key) ?: return
+
+            if (suggestion.value.size == 1) {
+                //It was decided to only handle the case where there is only one notification per configuration scope
```

Comment:
Only first a first baby step, probably we should revisit this at the end of the sprint or during a hardening